docs/src/components/Sandbox/Sandbox.less

```diff
@@ -0,0 +1,86 @@
+@import (reference) "~seek-style-guide/theme";
+
+@sandbox-background: darken(@sk-blue, 13);
+@sandbox-preview-background: darken(@sandbox-background, 5);
+@sandbox-line-height: (@row-height * 5);
+@sandbox-marker-size: (@gutter-width / 2);
+@sandbox-marker-color: fade(lighten(@sk-blue, 50), 50);
+@sandbox-editor-height: (@row-height * 47);
```

Comment:
What are your thoughts about making this a percentage? Reason is i tried to zoom out to get more code space and see more previews at once and the preview area grows but the editor area is locked to 47 grids rows.

Reply:
Yeah, good idea. I'll look at it.

Reply:
Done 👌